### ISSUE TITLE
CLI error in Windows

### ISSUE THREAD

**Issue by ACXgit** [tivac/mithril-objectify#17]
Hi tivac,
I had some issues running mithril-objectify via CLI in Windows.

First I had to install it globally via `npm install -g mithril-objectify` to make it being recognized as command (is there another way to run it only locally inside the project?).

Then to make it work I had to change the `mithril-objectify.cmd` content from:
```
"%~dp0\node_modules\mithril-objectify\bin\cli.js"   %*
```
to
```
@IF EXIST "%~dp0\node.exe" (
  "%~dp0\node.exe"  "%~dp0\node_modules\mithril-objectify\bin\cli.js" %*
) ELSE (
  node  "%~dp0\node_modules\mithril-objectify\bin\cli.js" %*
)
```
Now it's working like a charm, I hope this could be useful.

**Comment by tivac** [comment]
It's because NPM doesn't install it correctly, but it's my fault. NPM checks for a shebang at the beginning of the bin file, but this project is missing one:

https://github.com/tivac/mithril-objectify/blob/master/bin/cli.js#L1

A correct bin file looks like this:

https://github.com/tivac/dullard/blob/master/bin/cli.js#L1

I'll add the missing shebang and get 1.2.1 published here shortly.

**Comment by tivac** [comment]
@ACXgit published `mithril-objectify@1.2.1` which should fix this issue.

**Comment by ACXgit** [comment]
Just tested and it works as expected :+1:
